packages/java/src/browser/java-commands.ts

```diff
@@ -107,6 +122,18 @@ export class JavaCommandContribution implements CommandContribution, MenuContrib
             },
             isEnabled: () => this.javaClientContribution.running
         });
+        commands.registerCommand(JAVA_IGNORE_INCOMPLETE_CLASSPATH, {
+            execute: async () => {
+                await this.preferencesService.set('java.errors.incompleteClasspath.severity', 'ignore');
```

Comment:
It sets as a user setting. Is it expected?

Reply:
Looks like expected: https://github.com/redhat-developer/vscode-java/blob/b6b922cedc2d0c7e859aedb4792427e9b4ba52a4/src/extension.ts#L393